tripy/tripy/flat_ir/ops/base.py

```diff
@@ -101,3 +102,102 @@ def _op_name(self) -> str:
             The name of this operation.
         """
         return self.__class__.__name__
+
+
+class FlatIRFunction(abc.ABC):
```

Comment:
Doesn't seem to be an abstract class:
```suggestion
class FlatIRFunction:
```

Reply:
Might also be worth having this in a separate file